<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="windowTitle">
   <string>Форма логина</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_username">
     <property name="text">
      <string>Имя пользователя:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="usernameField">
     <property name="placeholderText">
      <string>Введите имя пользователя</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_password">
     <property name="text">
      <string>Пароль:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordField">
     <property name="placeholderText">
      <string>Введите пароль</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="loginButton">
     <property name="text">
      <string>Войти</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

